CONTRIBUTING.md

```diff
@@ -67,3 +67,71 @@ squashed one:
 1. Do not end the subject line with a period
 1. Use the imperative mood in the subject line
 1. Use the body to explain what and why vs. how  
+
+## Contribution Tutorial
```

Comment:
We're wondering whether it would maybe better add this tutorial to some google doc we could share or we link to this document on your branch. Our concern is that these instructions could go out of date if git changes things. As it is a standard action done on git, developers should generally either know it already or be able to google it. Hence, there is probably no need to add it to our own documenation.

Reply:
Would it be better to describe what actions are being done and link to GitHub's or Git-SCM's own documentation about how to perform those actions? Some contributors are relatively new to using Git and having some space for Git specific instructions would help facilitate their onboarding (plus future contributors) to making contributions, I believe.

Also, the tutorial should include our project / context specific instructions such as the style guide already part of this doc, CONTRIBUTING.md as well as "you must do this before making a pull request" instructions from the [SETUP.md ](https://github.com/rtcharity/eahub.org/blob/default/SETUP.md) Formatting section, yes?

Should we duplicate or move the Formatting section of SETUP.md to the contribution tutorial so that when making a pull request, one doesn't have to check multiple pages?

I worry that making a google doc spreads the documentation further afield and makes a contributor check even more things to get started, and would have the same issues of potentially going out of date if git changes things too.